### ISSUE TITLE
Error when attempting to run script

### ISSUE THREAD

**Issue by Malibu-Stacey** [Loathing-Associates-Scripting-Society/garbage-collector#2]
```
> call scripts\garbage-collector\garbo.js

Using adventure value 5000.
Collecting garbage!

Autoattack is disabled.
Preference currentMood changed from default to apathetic
Preference customCombatScript changed from WHAM to garbo
CCS set to garbo
JavaScript exception: No stashClan set. (file:/D:/KoLmafia1/scripts/garbage-collector/garbo.js#5153)
at file:/D:/KoLmafia1/scripts/garbage-collector/garbo.js:5153 (StashManager)
at file:/D:/KoLmafia1/scripts/garbage-collector/garbo.js:5122 (withStash)
at file:/D:/KoLmafia1/scripts/garbage-collector/garbo.js:4016 (main)
```

**Comment by Malibu-Stacey** [comment]
also fixed in #4

**Comment by Malibu-Stacey** [comment]
> also fixed in #4

this person is a liar.
